packages/ui/src/components/va-select/VaSelect.vue

```diff
@@ -309,8 +309,12 @@ export default defineComponent({
         return value
       },
 
-      set (value: any) {
-        emit('update:modelValue', getValue(value))
+      set (value: SelectableOption | SelectableOption[]) {
+        if (Array.isArray(value)) {
+          emit('update:modelValue', value.map(v => getValue(v)))
```

Comment:
If `getValue` doesn't accept any other arguments, you can just provide it as callback.